### DISCUSSION TITLE
点击登录后显示404  File not found

### DISCUSSION THREAD

**Discussion by mcthp** [gitalk/gitalk#491] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/35915872/143475571-491670e9-2034-4be7-817b-aa41a6890a18.png)
一点login就会显示(第一次点还会跳转至github的授权页,后来没了)
![image](https://user-images.githubusercontent.com/35915872/143475645-f2114c38-3fae-404f-a04c-1342cb3713a3.png)
求解

**Comment by SilenceEvans** [comment, marked as answer]
你如果用的是next主题的话，检查一下主题下_config.yml文件中关于gitalk配置部分里面各个参数：仓库名、Cilend_ID等正不正确，另外可能就是你的文章标题中有类似于中文破折号、中文括号这样的符号，因为有这些的话，你点击登录时那个url也会有这些符号，可能会发生url转义的错误，我的就是因为文章标题中有这些导致的，但你的不太像，那很可能就是是因为配置参数还是有问题，希望能帮助到你
